backend/test/fixtures/enrollments.yml

```diff
@@ -1,73 +1,40 @@
 DEFAULTS: &DEFAULTS
-  target_api: "franceconnect"
-  previous_enrollment_id: null
-  copied_from_enrollment_id: null
+  siret: "83906810300010"
```

Comment:
submited, validated and refused enrollment should be completed. Plz fill all the missing fields.